jstests/core/profile_sampling.js

```diff
@@ -0,0 +1,48 @@
+// special db for running tests in parallel
+var stddb = db;
```

Comment:
I was copying the pattern in, e.g., `jstests/core/profile1.js`, which restores the value of `db` at the end of the file. Is that not necessary?